### PR TITLE
Create enum for build info constants

### PR DESCRIPTION
**Pull request by booneng**
Create enum for build info constants so we have a consistent way of accessing the build info properties.

---
This change is [Reviewable](https://reviewable.io/reviews/google/mobly/778)

### CODE REVIEW

**Review by eric100lin — CHANGES_REQUESTED** [review]
> *[Reviewable](https://reviewable.io/reviews/google/mobly/778)* status: 0 of 1 files reviewed, 1 unresolved discussion (waiting on @booneng and @xpconanfan)

___
*[mobly/controllers/android_device.py, line 771 at r1](https://reviewable.io/reviews/google/mobly/778#-MrQc9OvAv-Q_lP5UQqe:-MrQc9OvAv-Q_lP5UQqf:bkc949s) ([raw file](https://github.com/google/mobly/blob/8912a13fa022a706f7776202cea845a0bf9ddbb6/mobly/controllers/android_device.py#L771)):*
> ```Python
>   @property
>   def build_info(self):
>     """Get the build info of this Android device, including build id and type.
> ```

https://google.github.io/styleguide/pyguide.html#383-functions-and-methods
The docstring should be descriptive-style ("""Fetches rows from a Bigtable.""") rather than imperative-style ("""Fetch rows from a Bigtable.""").



_Code quote:_
```Python
Get
```

**Review by booneng — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/google/mobly/778)* status: 0 of 1 files reviewed, 1 unresolved discussion (waiting on @eric100lin and @xpconanfan)

___
*[mobly/controllers/android_device.py, line 771 at r1](https://reviewable.io/reviews/google/mobly/778#-MrQc9OvAv-Q_lP5UQqe:-MrQtEgd0iY5MjFRZe6V:b-896fix) ([raw file](https://github.com/google/mobly/blob/8912a13fa022a706f7776202cea845a0bf9ddbb6/mobly/controllers/android_device.py#L771)):*
<details><summary><i>Previously, eric100lin (Eric Lin (Tzu Hsiang Lin)) wrote…</i></summary><blockquote>

https://google.github.io/styleguide/pyguide.html#383-functions-and-methods
The docstring should be descriptive-style ("""Fetches rows from a Bigtable.""") rather than imperative-style ("""Fetch rows from a Bigtable.""").
</blockquote></details>

Done.

File: mobly/controllers/android_device.py [hunk 1]
```diff
@@ -430,6 +431,22 @@ def take_br(test_name, begin_time, ad, destination):
   utils.concurrent_exec(take_br, args)
 
 
+class BuildInfoConstants(enum.Enum):
+  """Enums for build info constants used for AndroidDevice build info."""
+
+  BUILD_ID = 'build_id'
```
Comment (xpconanfan) [inline mobly/controllers/android_device.py:437]: can we contain all the info needed in this dataclass so we just need to do a loop in the `build_info` property?

**Review by xpconanfan — CHANGES_REQUESTED** [review]

**Review by booneng — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/google/mobly/778)* status: 0 of 1 files reviewed, 2 unresolved discussions (waiting on @booneng, @eric100lin, and @xpconanfan)

___
*[mobly/controllers/android_device.py, line 437 at r2](https://reviewable.io/reviews/google/mobly/778#-Mr1YvmvmoaJjQc7v9LQ-r2-437:-MrV2qwp5ykrQGjhf2Bi:b-896fix) ([raw file](https://github.com/google/mobly/blob/72bfcf709cb7f7ae53b7da228983c61562b21e42/mobly/controllers/android_device.py#L437)):*
<details><summary><i>Previously, xpconanfan (Ang Li) wrote…</i></summary><blockquote>

can we contain all the info needed in this dataclass so we just need to do a loop in the `build_info` property?
</blockquote></details>

Done.

**Review by eric100lin — APPROVED** [review]
:lgtm:

> Reviewed 1 of 1 files at r3, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/google/mobly/778#-:-MrVIfgt4HjbtLa03td2:bnfp4nl)* status: all files reviewed, 2 unresolved discussions (waiting on @eric100lin and @xpconanfan)

**Review by xpconanfan — APPROVED** [review]

**Pull request merged**